### ISSUE TITLE
The popup picker steals focus when it closes

### ISSUE THREAD

**Issue by plamen89** [joeattardi/picmo#229]
**How to reproduce:**

1. Open a popup picker
2. Click inside a text input or textarea

**Expected behaviour:**
- The text input/textarea gains focus and the popup picker closes

**Actual behaviour:**
- The text input/textarea gains focus for a few miliseconds, but then loses it when the popup picker closes

Pseudo code (since I can't get jsfiddle to work with this library for some reason):

HTML
```
<input type="text" />

<button id="emoji-btn" onclick="toggle()">
  Open picker
</button>
```

Javascript
```
function toggle() {
	picker.toggle();
}

const picker = createPopup({}, {
  referenceElement: document.getElementById("emoji-btn"),
  triggerElement: document.getElementById("emoji-btn")
});
```

**Comment by joeattardi** [comment]
Currently when the popup closes, it returns focus to the trigger. If you clicked on something else, that's why it steals focus.

I have fixed this issue by changing the popup picker to only return focus to the trigger if an emoji was selected or Escape was pressed.

Clicking on other elements outside the picker should no longer steal focus.

This will be included in the next release.

**Comment by plamen89** [comment]
Thank you for the fast response. Any idea when the next release will be available?

**Comment by joeattardi** [comment]
Hoping for later this week, just working out some styling issues. I will post in the announcements when it's out.

**Comment by joeattardi** [comment]
Just published 5.5.1 that should include this fix.
